app/views/content/help-and-support.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Help and support on getting into teaching"
+heading: "Get help and support"
+image: "media/images/content/hero-images/0025.jpg"
+title_paragraph: We're here to answer your questions and provide advice about getting into teaching, whether you're just thinking about teaching or you're ready to apply.
+description: |-
+  Get help and support on getting into teaching by talking to a dedicated adviser, going to a teacher training event, or registering for relevant email updates.
```

Comment:
```suggestion
  Get help and support on getting into teaching by talking to a dedicated adviser, going to a teacher training event, or registering for tailored emails.
```
or 'personalised guidance by email'?

I think 'relevant' is superfluous - who would register for irrelevant emails? Plus updates is vague and confusing?

@EllieNodder